financial-templates-lib/test/clients/ExpiringMultiPartyClient.js

```diff
@@ -167,7 +166,7 @@ contract("ExpiringMultiPartyClient.js", function(accounts) {
 
     await updateAndVerify(
       client,
-      [sponsor1, sponsor2],
+      [sponsor1],
```

Comment:
this test was actually **wrong** I believe. The sponsor is liquidated and so the address should _not_ be in the client any more when calling `client.getAllSponsors()` in `updateAndVerify`